curr_scraper/apps/api_caller/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Currency(models.Model):
+    code = models.TextField()
+    description = models.TextField()
+
+    class Meta:
+        app_label = "api_caller"
+
+
+class CurrencyRateRecord(models.Model):
+    timestamp = models.TimeField()
```

Comment:
TimeField does not represent timestamp at all, it stores time independent of any particular day (https://docs.python.org/3/library/datetime.html#available-types).
You need something like that:
```
timestamp = models.DateTimeField(auto_now_add=True)
```